tailwind.config.js

```diff
@@ -61,6 +61,7 @@ module.exports = {
         7.15:"1.7875rem",
         7.5:"1.875rem",
         7.83: "1.958rem",
+        9: "2.25rem",
```

Comment:
By default, we do have this line-height value in Tailwind, so you don't need to add it here.